docs/install/prerequisites.md

```diff
@@ -39,3 +39,13 @@ capabilities:
 
 [libbpf CO-RE documentation]: https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere
 [BTFHUB]: https://github.com/aquasecurity/btfhub-archive
+
+# Corner Cases
+
+## Drop Capabilities Error
+
+Tracee-ebpf and tracee-rules both try to reduce capabilities upon startup.
+Some environments won't allow capabilities dropping because of permission issues.
+Failure in capabilities dropping will result tracee's crash with a matching error, to guarantee that tracee isn't running with excess capabilities without the user agreement.
+To allow tracee to run with high capabilities and prevent crushing, the `--allow-high-capabilities` flag can be used.
+For docker users, to allow tracee-ebpf high capabilities the environment variable ` EBPF_ALLOW_HIGH_CAPABILITIES=1` should be used.
```

Comment:
I think the env flag should affect both programs, so maybe just `ALLOW_HIGH_CAPABILITIES`, otherwise you would need to run the image both with the env flag and the `--ahc` flag itself following which isn't very intuitive UX.
And a nit: there's a space at the start of `EBPF_...`

Reply:
The problem is, you won't be able to use the `--allow-high-capabilities` flag in the container, which reduce usabilitiy.
However, I think it might be better either way.
I need to check if I can make it so that it will use the flag with the env var only if it is defined.